### ISSUE TITLE
[HOLD] Split - Split preview shows 0 instead of "Your split $0.00" when split amount is 0.00

### ISSUE THREAD

**Issue by lanitochka17** [Expensify/App#41752]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 1.4.71-0
**Reproducible in staging?:** Y
**Reproducible in production?:** N
**If this was caught during regression testing, add the test name, ID and link from TestRail:** N/A
**Issue reported by:** Applause - Internal Team

## Action Performed:
1. Go to staging.new.expensify.com
2. Go to group chat
3. Go to + > Split expense
4. Enter amount > Next
5. Enter 0.00 amount on the first person (split creator)
6. Split the expense
7. Click on the split preview

## Expected Result:
In Step 6, the expense preview will show "Your split $0.00"
In Step 7, the split creator under "Split amount" will show $0.00, or the split creator will not show up under the list

## Actual Result:
In Step 6, the expense preview shows "0"
In step 7, the split creator shows up under "Split amount" list with no split amount (0.00) indicated

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Native
- [x] Android: mWeb Chrome
- [ ] iOS: Native
- [ ] iOS: mWeb Safari
- [x] MacOS: Chrome / Safari
- [x] MacOS: Desktop

## Screenshots/Videos

Add any screenshot/video evidence

</details> 

https://github.com/Expensify/App/assets/78819774/1280dc24-e9c5-4c2b-a450-ba3584568603

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by Nodebrute** [comment]
## Proposal

### Please re-state the problem that we are trying to solve in this issue.
Split preview shows 0 instead of "Your split $0.00" when split amount is 0.00
### What is the root cause of that problem?
`splitShare` will be 0 so instead of rendering next component react will render 0. 
https://github.com/Expensify/App/blob/9cad71513fc52ab0cfd5ca5a5434770eb6eec7a7/src/components/ReportActionItem/MoneyRequestPreview/MoneyRequestPreviewContent.tsx#L316
### What changes do you think we should make in order to solve the problem?

We should use a boolean value here. For example we can use isBillSplit
https://github.com/Expensify/App/blob/9cad71513fc52ab0cfd5ca5a5434770eb6eec7a7/src/components/ReportActionItem/MoneyRequestPreview/MoneyRequestPreviewContent.tsx#L316
### What alternative solutions did you explore? (Optional)

**Comment by marcaaron** [comment]
@youssef-lr Another one related to Splits. Off the top of my head it doesn't make a whole lot of sense to be able to "split $0" and have it say:

>"You split $0.00"

So, curious if we got these steps right. Either way, new feature not a deploy blocker IMO.

**Comment by youssef-lr** [comment]
I don't think it makes sense for someone to be the payer of a split and set the amount `0` for themselves. So this seems like we should show an error in this case. I'll bring this up in Slack.

**Comment by youssef-lr** [comment]
Will start a discussion today.

**Comment by youssef-lr** [comment]
Haven't got to this yet, will try tomorrow

**Comment by youssef-lr** [comment]
No update yet

**Comment by youssef-lr** [comment]
No update yet, we have paused working on vip-split.

**Comment by youssef-lr** [comment]
No update yet

**Comment by youssef-lr** [comment]
No update, I can't prioritize this now given vip-split is paused. Putting on monthly.

**Comment by youssef-lr** [comment]
Still paused.

**Comment by youssef-lr** [comment]
Same

**Comment by youssef-lr** [comment]
Same

**Comment by youssef-lr** [comment]
On hold.

**Comment by youssef-lr** [comment]
Same.

**Comment by youssef-lr** [comment]
Same

**Comment by youssef-lr** [comment]
Same

**Comment by youssef-lr** [comment]
Same.

**Comment by youssef-lr** [comment]
Closing https://github.com/Expensify/Expensify/issues/373470#issuecomment-2963780768
